plantcv/plantcv/threshold/threshold_methods.py

```diff
@@ -48,7 +46,7 @@ def binary(gray_img, threshold, max_value, object_type="light"):
     params.device += 1
 
     # Threshold the image
-    bin_img = _call_threshold(gray_img, threshold, max_value, threshold_method, "_binary_threshold_")
+    bin_img = _call_threshold(gray_img, threshold, 255, threshold_method, "_binary_threshold_")
```

Comment:
happy to update this to keep `max_value` as a variable here and set that to `255` above in the function if that's preferred

Reply:
Hmmm @nfahlgren do we want to remove the argument from the helper function too? I don't imagine we'd ever set it to anything else...